Dockerfile

```diff
@@ -6,6 +6,12 @@
 
 ARG TOOLCHAIN
 
+FROM fedora:39 AS zfs
+COPY hack/zfs-generate.sh /
+RUN dnf install -y zfs-fuse && rm -rf /var/cache/dnf
+# Needs to access FUSE
+RUN --security=insecure /zfs-generate.sh
```

Comment:
This file's header tells it's not to be modified, however I didn't find a way to configure Kres and make this work without editing Dockerfile

Reply:
I can help with that, kres is not easy :sweat_smile:

Reply:
Thanks! We can do it later when Andrey reviews the code change and the way I chose to test this

Reply:
that'd be a bit of a mess to setup in Kres, idk, in the worst case we can do a dumb simple separate repo which produces an image of ZFS filesystem